pages/api/auth/[...nextauth].ts

```diff
@@ -11,6 +14,14 @@ export default NextAuth({
             server: process.env.EMAIL_SERVER,
             from: process.env.EMAIL_FROM,
         }),
+         GithubProvider({
+            clientId: process.env.GITHUB_ID,
```

Comment:
Sweep: Let's fix the build issue. Possible solution is checking `process.env.GITHUB_ID` before adding to the providers.

Reply:
I just made a PR with check conditions